tslib/src/caching/redis-service.ts

```diff
@@ -40,7 +40,7 @@ export interface IRedisService {
    * @param {Array<string>} keys an array of keys to invalidate
    * @returns {Promise<Array<[error: Error | null, result: unknown]> | null | undefined>}
```

Comment:
The jsdoc is out of date now the call sig has changed.   Please consider either updating or deleting the jsdoc.